File: core_library/src/main/java/core/view/DisabledSwipeToRefresh.java
package core.view;

import android.content.Context;
import android.support.v4.widget.SwipeRefreshLayout;
import android.util.AttributeSet;

public class DisabledSwipeToRefresh extends SwipeRefreshLayout {
    public DisabledSwipeToRefresh(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    public boolean isEnabled() {
        return false;
    }
}

File: core_library_kt/src/main/java/core_kt/view/DisabledSwipeToRefresh.kt
package core_kt.view

import android.content.Context
import android.util.AttributeSet
import androidx.swiperefreshlayout.widget.SwipeRefreshLayout

class DisabledSwipeToRefresh(context: Context, attrs: AttributeSet) : SwipeRefreshLayout(context, attrs) {

    override fun isEnabled(): Boolean {
        return false
    }
}